wagn/lib/wagn/tasks/test.rake

```diff
@@ -74,7 +74,7 @@ namespace :test do
     ActiveRecord::Base.establish_connection
     tables.each do |table_name|
       i = "000"
-      File.open("#{Wagn.gem_root}/test/fixtures/#{table_name}.yml", 'w') do |file|
+      File.open("#{Cardio.gem_root}/test/fixtures/#{table_name}.yml", 'w') do |file|
         data = ActiveRecord::Base.connection.select_all(sql % table_name)
         file.write data.inject({}) { |hash, record|
           record['trash'] = false if record.has_key? 'trash'
```

Comment:
Ah. I think it's fair to say that code needs an update.

It's basically legacy code from the pre-gem days when changing from working on one site to another would generally mean editing config files.  Lewis created code shortcuts for this and used the trick for populating the template database, but now there are better ways!

Afaict, it's only used in generating fixtures at the moment, but still... 

Let's move the conversation off this PR, but good point!